src/cavendish_particle_tracks/_set_fiducial_dialog.py

```diff
@@ -0,0 +1,536 @@
+import napari
+import numpy as np
+from qtpy.QtWidgets import (
+    QAbstractItemView,
+    QComboBox,
+    QDialog,
+    QDialogButtonBox,
+    QGridLayout,
+    QHBoxLayout,
+    QLabel,
+    QPushButton,
+    QTableWidget,
+    QTableWidgetItem,
+    QVBoxLayout,
+)
+
+from ._analysis import FIDUCIAL_BACK, FIDUCIAL_FRONT, Fiducial
+from ._calculate import (
+    corrected_length,
+    depth,
+    magnification,
+)
+
+
+class Set_Fiducial_Dialog(QDialog):
```

Comment:
Should this dialog substitute the stereoshift_dialog?

Reply:
Was intended to be that it would substitute both this and magnification. Now that it's likely no longer doing that the changes made can probably be merged back into stereoshiftdialog.